domains/gunman.json

```diff
@@ -5,6 +5,6 @@
     "discord": "1115658967012626542"
   },
   "record": {
-    "A": ["216.24.57.1"]
+    "A": ["76.76.21.21"]
```

Comment:
Your website doesn't seem an incomplete template, i see "lorem ipsum" strings

Reply:
I am currently developing it and I need a text to fill it so I used lorem ipsum.